__tests__/app.test.js

```diff
@@ -12,29 +12,45 @@ beforeEach(() => {
 
   describe('GET /api/topics', () => {
       describe('Functionality', () => {
-        test('Returns an array of topic objects with slug and decription properties', () => {
-        return request(app)
+        test('Returns an array of topic objects with slug and decription properties', async () => {
+        const response = await request(app)
         .get("/api/topics")
         .expect(200)
-        .then((res) => {
-            expect(res.body.topics.length).toBe(3);
-            res.body.topics.forEach(topic => {
+            expect(response.body.topics.length).toBe(3);
+            response.body.topics.forEach(topic => {
                 expect.objectContaining({
                     slug: expect.any(String),
                     description: expect.any(String),
                   });
             })
-        })
+        
       })
     })
       describe('Error Handling', () => {
-        test('Returns 404 not found if no topics are found', () => {
-            return request(app)
+        test('Returns 404 not found if no topics are found', async () => {
+            const response = await request(app)
```

Comment:
Same as above

Reply:
So are you recommending i dont use async/await for the tests ? or is there just  a different way of using the return?